__tests__/app.test.js

```diff
@@ -43,4 +43,31 @@ describe('app', () => {
               expect(topic).toHaveProperty('description')  
           })
         })
-      })         
\ No newline at end of file
+      })    
+      
+      describe('GET /api/articles', () => {
+        test('should return a status :200 ',() => {
+          return request(app)
+          .get('/api/articles')
+          .expect(200)
+          .then(({body}) => {
+            expect(true).toEqual(true);        
+          });
+        });
+      })
```

Comment:
Make sure you're failing your tests first - is there a situation where expect(true).toEqual(true) would fail? Remember we're looking for the specifics of what is returned on the body here so log it out and see what data we want to look at. Arguably this could just be a single test rather than 2 separate ones. 

Try using the syntax for objectContaining below so that we can check the key value pairs rather than just the values. eg:
expect(article).toEqual(
            expect.objectContaining({
              author: expect.any(String)...etc

Reply:
The expect(true)toEqual(true) was the first test I did to check that the status 200 was working. I will remove and come up with new expects as suggested and implement the objectContaining method. Thanks.

Reply:
Will also merge the tests for task3 into a single one to avoid testing for true for true's sake like I did earlier on.  Thanks again.